### ISSUE TITLE
simplified InterfaceWithMatlab, part 2

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#1936]
Continuation of #1637

**Comment by dcwhite** [comment]
Still waiting on more use cases from users.

**Comment by dcwhite** [comment]
@jab0707 Are you using this feature or know anyone who is?

**Comment by jab0707** [comment]
No, I have been using the direct python interface for any of these related needs so far. I am not sure if anyone else is using this at the moment. I can check in our group.

**Comment by dcwhite** [comment]
**Analysis (2026-06-13)**

Continuation of #1637. Most items are syntax/parser improvements for the embedded MATLAB-syntax transpiler. Blocked on the same dependency chain as #1221: pip/SSL (#1295) must work before the MATLAB engine can be installed.

Additionally, comment thread notes no current active users — @jab0707 confirmed their group uses the direct Python interface instead. Suggest holding this until there's user demand.
